File: Kotlin/entradaSaida/Exemplo1.kt
package entradaSaida

fun main() {
    print("Digite a quantidade de km andados: ")
    val qtdKm = readln().toDouble()
    print("Digite a quantidade de horas: ")
    val qtdHoras = readln().toDouble()

    val media = qtdKm / qtdHoras

    println("A média de quilômetros é $media km/h")
}

File: Ruby_/entradaSaida/Exemplo1.rb
# frozen_string_literal: true

# em ruby, print não pula a linha
print "Digite a quantidade de km andados: "
km = gets.to_f # faz o input e transforma o resultado em float
print "Digite a quantidade de horas: "
qtdHoras = gets.to_f

media = km / qtdHoras

puts "A média de quilômetros é #{media}km/h"